packages/block-editor/src/components/editor-skeleton/index.js

```diff
@@ -18,10 +18,16 @@ function EditorRegions( {
 	className,
 } ) {
 	return (
-		<div className={ classnames( className, 'edit-post-editor-regions' ) }>
+		<div
+			className={ classnames(
+				className,
+				'block-editor-editor-skeleton',
+				'edit-post-editor-regions'
```

Comment:
I think we can drop this className tbh. Did you leave it for BC? This seems to be ready to land before beta1 so there shouldn't be any problem to remove the classname.

Reply:
Yeah, I left it in for backwards compat. I know of at least [one instance in Calypso](https://github.com/Automattic/wp-calypso/blob/0fe9f9d07ba498db6b1bc89fc3aeb767a84be392/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.scss#L19) where we're referring to this class name, and since we were having quite some trouble with class name changes in the past (specifically with the `.editor-` -> `.block-editor-` change, see https://github.com/Automattic/wp-calypso/pull/38857, #14420, and #19050), I wanted at least to give a grace period before dropping the old class name.

(Then again, a point can be made that this was never part of a public interface, and that that grace period didn't help in updating the affected Calypso selectors in time before they were dropped in the past either, so :man_shrugging: )